editor/src/components/editor/server.ts

```diff
@@ -176,6 +184,8 @@ export async function loadProject(
     return response.json()
   } else if (response.status === 404) {
     return { type: 'ProjectNotFound' }
+  } else if (response.status === 401) {
```

Comment:
btw I just realized we've been using 401s in the validators and now in `internal.projects.$id.access.spec.ts`, while this should actually be a `403`. We can fix the validator later, but can we please use 403s in this PR please?

Reply:
We should also handle the 401 in a different way. This comes back to what Malte was saying about how the current 404 screen doesn't distinguish between signed-in and not-signed-in users.

Reply:
fixed by @ruggi